sphinxcontrib/needs/directives/needimport.py

```diff
@@ -120,15 +119,49 @@ def run(self):
         for key, need in needs_list.items():
             need["tags"] = need["tags"] + tags
 
-        template_location = os.path.join(os.path.dirname(os.path.abspath(__file__)), "needimport_template.rst")
-        with open(template_location, "r") as template_file:
-            template_content = template_file.read()
-        template = Template(template_content)
-        content = template.render(needs_list=needs_list, hide=hide, id_prefix=id_prefix)
-        self.state_machine.insert_input(content.split('\n'),
-                                        self.state_machine.document.attributes['source'])
+        # template_location = os.path.join(os.path.dirname(os.path.abspath(__file__)), "needimport_template.rst")
+        # with open(template_location, "r") as template_file:
+        #     template_content = template_file.read()
+        # template = Template(template_content)
+        # content = template.render(needs_list=needs_list, hide=hide, id_prefix=id_prefix)
+        # self.state_machine.insert_input(content.split('\n'),
+        #                                 self.state_machine.document.attributes['source'])
```

Comment:
if this is no longer used, should it just be deleted?

Reply:
Ohh yes.
Will add you next time as reviewer.
Thanks.